common/effect_localization/wc_perk_effects.txt

```diff
@@ -8,4 +8,8 @@ blood_haze_perk_effect = {
 
```

Comment:
I guess you would like to put the stunning check into this trigger to block activities like hunts.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117260479-2d989580-ae60-11eb-825e-d8ba87a6622e.png)

</details>